### ISSUE TITLE
i.dateFormat is not a function

### ISSUE THREAD

**Issue by LikeBtn** [xdan/datetimepicker#369]
Hi,

I am using jQuery 1.11.3.

Here is the code:

        jQuery("#my_input").datetimepicker({
            startDate: '26.12.2015 08:31'
        });

Here the error I am getting:

	TypeError: Date.parseDate is not a function
	...ate&&t.isValidDate(e))return e;var a=e?Date.parseDate(e,w.formatDate):t.now(!0);...
	jquery....r (line 1, col 29523)

	TypeError: i.dateFormat is not a function
	...isabled"):-1!==w.disabledDates.indexOf(i.dateFormat(w.formatDate))?p.push("xdsof...
	jquery....r (line 2, col 647)

**Comment by LikeBtn** [comment]
With non-minimized plugin:

	TypeError: Date.parseDate is not a function
	var currentTime = sDate ? Date.parseDate(sDate, options.formatDate) : _this.now(...
	jquery....r (line 1437, col 32)
	
	TypeError: start.dateFormat is not a function
	...f (options.disabledDates.indexOf(start.dateFormat(options.formatDate)) !== -1) 
	jquery....r (line 1630, col 49)

**Comment by LikeBtn** [comment]
JavaScript Date object does not have a Date.parseDate method:

	Date.parseDate
	undefined
	Date.parse
	parse()

Does this plugin require some other plugins to function?

**Comment by winternet-studio** [comment]
Having the same issue. Any solution?

**Comment by LikeBtn** [comment]
You need to add to your JavaScript a Parse and Format Library. You can find it on http://xdsoft.net/jqplugins/datetimepicker/.

**Comment by thekrotek** [comment]
No, there're no Parse and Format Library on provided address. And documentation itself is outdated, it even contains old definition of locale.

Previous version (2.3.4) works like a charm and doesn't require any additional libraries. Will stick to it.

**Comment by mstephens** [comment]
@thekrotek Same here. Ended up using the [Bootstrap 3 date/time picker](https://eonasdan.github.io/bootstrap-datetimepicker/#minimum-setup) :)

**Comment by afro1114** [comment]
you need to use the full version of the plugin, it includes the dateformat functions 
https://github.com/xdan/datetimepicker/blob/master/build/jquery.datetimepicker.full.min.js

**Comment by j-schumann** [comment]
The CDN currently does **not** provide the full version: https://cdnjs.com/libraries/jquery-datetimepicker

**Comment by leonardofaria** [comment]
@LikeBtn, like @j-schumann said, you should download the full version.

Or you can use the minified version + moment.js from a CDN.
Check this example: https://jsfiddle.net/leonardofaria/b78ykkru/

**Comment by zhanghaowx** [comment]
@leonardofaria The date format is not fully compatible, at least i is minute in datetimepicker but m in moment.js. So to fix the display in timepicker, you need:
```
Date.prototype.dateFormat = function (format) {
    if (format == "H:i")
        return moment(this).format("HH:mm");
    return moment(this).format(format);
};
```

**Comment by idailylife** [comment]
You may use `jquery.datetimepicker.full.min.js` instead of `jquery.datetimepicker.min.js`. This works for me.

**Comment by dleffler** [comment]
I don't think the DateFormater object is exposed correctly to be called (I can't get it to work in minimized or non-minimized variations)
